Docs: Integrating with VTEX Headless CMS (#1881)
vtex/faststore: apps/site/pages/docs/headless-cms-integration/1-configuring-the-vtex-account.mdx

</Callout>

> ```bash copy

Reviewer: For some reason, a couple code lines has this extra indentation at the beginning for the line. Don't know if it's related to the indentation on the `mdx` file. 🫠
<img width="958" alt="image" src="https://github.com/vtex/faststore/assets/11613011/add7f990-71a5-4edd-badc-37b3650a6558">

Let's configure the URLs of the webhooks used by the VTEX Headless CMS app.

> 1. In the VTEX Admin, access **Extensions Hub > App management**

Author: Reminder: need to review this step after the new Headless CMS UI.

Install the VTEX Headless CMS dependencies in your VTEX account by running the following command:

```bash copy

>  vtex install vtex.admin-cms-graphql@0.x vtex.admin-cms-graphql-rc@1.x 

Author: ```suggestion
 vtex install vtex.admin-cms@1.x
```

```

> ![installation-message](https://vtexhelp.vtexassets.com/assets/docs/src/cms-installing-dependencies___ad0f763129cfef77d56b4fdaba54be98.png)

Reviewer 2: O print mostra um comando diferente (nome do app)

  background.
</Callout>

> 5. Fill in the **Preview URL** filed with your production URL. This URL activates the button on VTEX Headless CMS for you to preview the changes made on a page. made_Replace the values between curly brackets according to your scenario._

Reviewer 2: ```suggestion
5. Fill in the **Preview URL** field with your production URL. This URL activates the button on VTEX Headless CMS for you to preview the changes made on a page. _Replace the values between curly brackets according to your scenario._
```

``` 
https://{account}.vtex.app/api/preview
```

> 6. Fill in the **Production base URL** filed with your store endpoint to preview changes in the Headless CMS before publishing them. _Replace the values between curly brackets according to your scenario._

Reviewer 2: ```suggestion
6. Fill in the **Production base URL** field with your store's endpoint to preview changes in the Headless CMS before publishing them. _Replace the values between curly brackets according to your scenario._
```
